lib/endpoint/find-description.js

```diff
@@ -5,8 +5,8 @@ function findDescription (state) {
     .filter(block => block.type === 'description')
     .map(block => block.text)
     .join('\n\n')
-
+  const newdescription = description.replace('/v3/', 'https://developer.github.com/v3/')
```

Comment:
this will break if a link is already absolute. It would also replace `/v3/` if it’s not a link URL. A better regex would be something like 

```js
description.replace('](/v3/', '](https://developer.github.com/v3/')
```

Reply:
also can you use camelCase for the variable name: `newDescription`

Reply:
Thanks let me update this change.

Reply:
I had a look at other links and there are also relative paths starting with `/apps` and `/webhooks` so make sure to catch these, too :)

Reply:
I don't understand this part would you please explain which one?

Reply:
Make sure that you also replace e.g. 

```
[web flow](/apps/building-integrations/setting-up-and-registering-oauth-apps/about-authorization-options-for-oauth-apps/)
```

with 

```
[web flow](https://developer.github.com/apps/building-integrations/setting-up-and-registering-oauth-apps/about-authorization-options-for-oauth-apps/)
```

You can test it with https://github.com/octokit/routes/blob/a509e1a89672025ec580405648b9c5666e3e7b06/routes/oauth-authorizations/create-a-new-authorization.json#L44

And make sure you replace e.g.

```
[ping event](/webhooks/#ping-event)
```

with 

```
[ping event](https://developer.github.com/webhooks/#ping-event)
```

You can test that with https://github.com/octokit/routes/blob/a509e1a89672025ec580405648b9c5666e3e7b06/routes/orgs/hooks/ping-a-hook.json#L20